Lexic.txt

```diff
@@ -0,0 +1,44 @@
+Alphabet:
+- Upper and lower case letters (A-Za-z)
+- underline character '_' and minus character'-';
+- decimal digits 
+
+Lexic
+a.Special symbols:
+- operators: + - * / < > = <= >= == // || && :
+- separators: () {} ; space
+- keywords: int char bool array let read write check else while return and or true false 

```

Comment:
You listed `||` and `&&` as logical operators, but have keywords `and` and `or` for the same operation as well.